src/features/chat/chat.ts

```
@@ -411,107 +423,175 @@ export class Chat {
+              break;
+
+            case "animation":
+              const animation = await loadVRMAnimation(`/animations/${data}`);
+              if (!animation) throw new Error("Loading animation failed");
+              this.viewer?.model?.playAnimation(animation, data);
+              requestAnimationFrame(() => this.viewer?.resetCameraLerp());
+              break;
+
+            case "playback": 
+              this.viewer?.startRecording();
+              const payload = JSON.parse(data);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)